apps/client/app/routes/logout.tsx

```diff
@@ -0,0 +1,10 @@
+import { ActionFunctionArgs, redirect } from "@remix-run/node";
+import { logout } from "~/services/auth.server";
+
+export async function loader() {
+  return redirect("/");
+}
+
+export async function action({ request }: ActionFunctionArgs) {
+  return logout(request);
```

Comment:
This logout need to remove or invalidate the session?

Reply:
The logout should remove the session I think